src/Components/Scenes/Governance/SensitivityTable.tsx

```diff
@@ -8,18 +8,21 @@ import {
   addSensitivityLevelsRequestFetch
 } from '../../../State/Governance/actions';
 import { TextField } from '@material-ui/core';
+import SnackBar from '../../Presentational/Modal/SnackBar';
 
 const SensitivityTable = ({ isModalOpen, setIsModalOpen }: any) => {
-  const { sensitivity, setSelectedRemoveRowId, isLoading } = useSelector(
-    (state: any) => state.governance
-  );
+  const {
+    sensitivity,
+    setSelectedRemoveRowId,
+    isLoading,
+    message
+  } = useSelector((state: any) => state.governance);
   const titleText = `Sensitivity Levels`;
```

Comment:
move titleText and columns to after the lines of useDispatch and before useEffect

Reply:
I will move it on my next PR